### PR TITLE
Domene

### PR DESCRIPTION
**Pull request by Domene99**
Change README to have portfolio link

### CODE REVIEW

File: portfolio/pom.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
```
Comment (ssmall) [inline portfolio/pom.xml:37]: nit: this TODO could be removed

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,53 @@
...
...bJ+CWW0ALNvMaW6a6qCl4gEi5az2W+gipxRbu8O5kkyzHBklaXRC1jLnDXcqGVVueiWirFIetu4HlPn+rZ3TBSVOVBnqiKi0jWpc2oRAuOYofDBgzHzCsjHdqm3Li6mzAP1oK+YKRNEDIXpR+gwHNnnm1DM3aua9iCHW/ai9ymkmxmoUDN5qR7w7eQgvcCR4mDnKLRyvJB+JHfpXe5AECxH1nmKfEIKdFEef7RfKPS/uof9i2GAfGSWrE7fz4+gFB7IyB2MBZYCLfYDLt7wXP5duSTPbUw/8nTR7y/rlClVbGYC0g6TEvjuza5024jF2B4ZhancfbJYtub0D2l9WZSyKqRwwUypkJczU2Bq8hxShGUUpk7l0wpMP5KlPzL9mmZxQEndkMnvlDK3CTAdXGTZYEZhaoHNs2MGKMyekdJh5Z915saM1HOlPWeCzSKewnSDnpng9CPZrmaRSpzCHeOFlsi18+DKnCdoi1QZCjNnKaPwJdyWcMkPEMatDjY6aiEdNoBLGQqjIoXP2t7DkssNdMr49Crwo7stcBoyClO1yVCYGSQphekV412XzX2M672tPuad+V5Bc7xF4eT2l67He53h+7d/5xEpwk5hjkaawqwViabwejI/PUVyxynr0quxUBWrrzfPSCdNYSYoMwozqzKj8OfgqpriKhZuKIBdEJ2D4aQWVCaa0pQnQ2EEKBypco9WfUI0j1gdj+l5X7mXIaAQaAslWJP4PQwEAdKJ3e8+rVkBhUDjq1C9QEwC0aA2o5dJ3jTmMgBmdUbhw2o7H2WJ48n7pjYPzgpGIKDw13Y+xa4tvKJuJr5NrDLOpdJqQXL3Pw6Awqw7IHBCceR8JL9q6x2UqyMZjW1zuxumRKQUHqrkyWl3rW680Ro7sQM7GHeq1KJYdXkSEiQtzPrytW3gHVPN075mdToSQuw1jhvoNWXEKLDRdAnpWI8ExXDQMcBlJKqRaQynpqfRjr41CxgEmwbPY4v8mcO+IbkaVKttbwQVy0k5WG7UNlV5LPa19uTbf2j7nviqaeanEWzbly4UIu/dnTSC0g2kuvHVdyoL17E/bd79qwWnb3TEHwvL74suC/s7OOw2gWxkbWfsbGNpafBfw2m428zCMUFoZxw439NjpdbfP4bF6pT8XL49847bUNVosKo/bI3gP9e/DWznbRTpAAAAAElFTkSuQmCC"
```
Comment (anniefu) [inline portfolio/src/main/webapp/index.html:22]: Whoa, I didn't even know you could do this. I would recommend just checking in the image you want to your repo and referencing it by relative path. Unless you don't want to check in the photo to github for whatever reason.

**Review by ssmall — APPROVED** [review]
LGTM! Just one nit.

File: portfolio/pom.xml (hunk 1 above)
Comment (Domene99) [reply portfolio/pom.xml:37]: Will do thanks

File: README.md [hunk 3]
```diff
@@ -1,13 +1,5 @@
...
-To get started:
-
-- Login to [Google Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor)
-- Clone this repo: `cd; git clone https://github.com/googleinterns/step.git`
-- Open the GitHub setup walkthrough: `teachme ~/step/walkthroughs/week-1-setup/github-setup-walkthrough.md`
-  - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
-
-Then follow the on-screen instructions to set up your repo.
+## Portfolio deployment link
+https://domene-step-2020.ue.r.appspot.com/
```
Comment (anniefu) [inline README.md:5]: Nice! I like the gifs :)

**Review by anniefu — COMMENTED** [review]
Haha, I'm late to the party here, but I'm gonna leave these comments anyway.

Also, I'd recommend using more descriptive PR titles in the future, just so reviewers know what to expect.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (Domene99) [reply portfolio/src/main/webapp/index.html:22]: Will do, I think I was just a bit lazy there

File: README.md (hunk 3 above)
Comment (Domene99) [reply README.md:5]: Thanks, they are really old projects which I felt would fit in here

**Pull request merged**